apps/site/src/app/(home)/sections/Landing/Landing.tsx

```diff
@@ -16,4 +39,68 @@ const Landing = () => {
 	);
 };
 
+const Fireflies = () => {
```

Comment:
Nitpick: since this takes up a lot of lines, we should extract this to its own component.